tests/kernel_performance/install_qatestset.pm

```diff
@@ -31,7 +31,7 @@ sub install_pkg {
     assert_script_run("cd /root/sleperf/SLEPerf; ./installer.sh common-infra");
 
     # Install qa_lib_ctcs2 package to fix dependency issue
-    zypper_call("install qa_lib_ctcs2");
+    zypper_call("install qa_lib_ctcs2 ntp");
```

Comment:
I remember ntp was installed by deployment.sh,why it install again here?

Reply:
I hope it is compatible with kinds of project to avoid missing depends package.